core/controllers/editor.py

```diff
@@ -76,15 +88,66 @@ class ExplorationHandler(EditorHandler):
     """Page with editor data for a single exploration."""
 
     GET_HANDLER_ERROR_RETURN_TYPE = feconf.HANDLER_TYPE_JSON
+    URL_PATH_ARGS_SCHEMAS = {
+        'exploration_id': {
+            'schema': {
+                'type': 'basestring'
+            }
+        }
+    }
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {
+            'v': {
+                'schema': {
+                    'type': 'int'
+                },
+                'default_value': None
+            },
+            'apply_draft': {
+                'schema': {
+                    'type': 'basestring'
+                },
+                'default_value': None
+            }
+        },
+        'PUT': {
+            'version': {
+                'schema': {
+                    'type': 'int'
+                }
+            },
+            'commit_message': {
+                'schema': {
+                    'type': 'basestring',
+                    'validators': [{
+                        'id': 'has_length_at_most',
+                        'max_value': constants.MAX_COMMIT_MESSAGE_LENGTH
+                    }]
+                },
+                'default_value': None
```

Comment:
I agree with you but this line only raise error when commit message is not none and it is exceeding max_size length.
You can take a look at [here](https://github.com/oppia/oppia/blob/b1846a839bfb72a5430031fcd2e27638004ad92b/core/domain/exp_services.py#L1090) or try to save the draft changes in an exploration without providing any commit message.
In case when I remove the default_value: None from the schema then commit message will become a mandatory field in ExplorationHandler but currently it is optional.
Looking forward for your point. 
Thank you.

Reply:
Thanks for the clarification.